### ISSUE TITLE
openAlarmsAndRemindersSettings android 14

### ISSUE THREAD

**Issue by BurtsevC** [Dev-hwang/flutter_foreground_task#262]
I checked openAlarmsAndRemindersSettings in Android 12 it works as it is (added to Alarms and Reminders). But in Android 14 it's not added to list and window is not showing.

**Comment by Dev-hwang** [comment]
@BurtsevC 

If the window does not appear, it means that the permission has been granted.

I confirmed that AlarmManager works well even without that app in the list.

I add a function to check whether permissions are granted before opening the window. I forgot 😅

**Comment by Dev-hwang** [comment]
@BurtsevC 

released `flutter_foreground_task: 8.5.0`

* [**FEAT**] Add `canScheduleExactAlarms` utility
  - Returns whether the `android.permission.SCHEDULE_EXACT_ALARM` permission is granted.
  - In some cases, permission is granted automatically. Don't worry :)

```dart
      // Use this utility only if you provide services that require long-term survival,
      // such as exact alarm service, healthcare service, or Bluetooth communication.
      //
      // This utility requires the "android.permission.SCHEDULE_EXACT_ALARM" permission.
      // Using this permission may make app distribution difficult due to Google policy.
      if (!await FlutterForegroundTask.canScheduleExactAlarms) {
        await FlutterForegroundTask.openAlarmsAndRemindersSettings();
      }
```

**Comment by BurtsevC** [comment]
@Dev-hwang 

Thanks a lot! Now it's clear))
